bridge/deploy/04_deploy_bridge.js

```diff
@@ -0,0 +1,29 @@
+module.exports = async function ({getNamedAccounts, deployments}) { // HardhatRuntimeEnvironment
+  const {deployer, bridgeProxy} = await getNamedAccounts();
+  const {deploy, log} = deployments;
+
+  if (bridgeProxy) return;
+
+  const deployResult = await deploy('Bridge', {
+    from: deployer,
+    log: true,
+  });
+  if (deployResult.newlyDeployed) {
+    log(`Contract Bridge deployed at ${deployResult.address} using ${deployResult.receipt.gasUsed.toString()} gas`);
+  }
+
+  // We need the multisig to update the contract
+  // const MultiSigWallet = await deployments.get('MultiSigWallet');
+  // const Bridge = await deployments.get('Bridge');
+};
+module.exports.id = 'deploy_bridge'; // id required to prevent reexecution
+module.exports.tags = ['Bridge', 'new'];
+// module.exports.dependencies = [
```

Comment:
We should remove the comment and set the right dependencies

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/267/commits/e557c97710c5051828bf529a1c4b1bb23f97709d